atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -330,7 +330,10 @@ def _get_email_from_koji_obj(self, obj):
         else:
             if not self.email_domain:
                 raise RuntimeError("Empty email_domain specified")
-            return '@'.join([obj['name'], self.email_domain])
+            elif 'name' not in obj:
+                raise RuntimeError("Koji object name is missing")
+            else:
```

Comment:
How this will prevent tracebacks in logs, when in `_get_receivers_list` this Exception will be caught and traceback printed via `log.exception()`

Reply:
Just change log.exception into log.info, as @rcerven proposed above